tukaan/widgets/tooltip.py

```diff
@@ -0,0 +1,36 @@
+from tukaan import Label
+
+class ToolTip():
+    def __init__(self, window, widget=None, orient="E", *args, **kwargs):
+        try:
+            self.text = kwargs.pop("text")
+        except:
+            raise Exception("No text provided for ToolTip")
+        self.window = window
+        self.widget = widget
+        self.orient=orient
```

Comment:
Black complained about this line, it likes to have spaces around operators.

If you're working in your local clone, you can `pip install black isort`, and run them on your files `black tukaan/*.py tukaan/*/*.py` (same with `isort`) to format them, this way the checks won't fail.